code/datums/status_effects/neutral.dm

```diff
@@ -391,42 +391,7 @@
 					do_sparks(5,FALSE,alt_clone)
 					alt_clone.emote("spin")
 					owner.emote("spin")
-					var/static/list/say_phrases = list(
-						"Bugger me, whats all this then?",
-						"Sacre bleu! Ou suis-je?!",
-						"I knew powering the station using a singularity engine would lead to something like this...",
-						"Wow, I can't believe in your universe Cencomm got rid of cloning.",
-						"WHAT IS HAPPENING?!",
-						"YOU'VE CREATED A TIME PARADOX!",
-						"You trying to steal my job?",
-						"So that's what I'd look like if I was ugly...",
-						"So, two alternate universe twins walk into a bar...",
-						"YOU'VE DOOMED THE TIMELINE!",
-						"Ruffle a cat once in a while!",
-						"I'm starting to get why no one wants to hang out with me.",
-						"Why haven't you gotten around to starting that band?!",
-						"No!! I was just about to greentext!",
-						"Kept you waiting huh?",
-						"Oh god I think I'm ODing I'm seeing a fake version of me.",
-						"Hey, I remember that phase, glad I grew out of it.",
-						"Keep going lets see if more of us show up.",
-						"I bet we can finally take the clown now.",
-						"LING DISGUISED AS ME!",
-						"El psy congroo.",
-						"At long last! My evil twin!",
-						"Keep going lets see if more of us show up.",
-						"No! Dark spirits, do not torment me with these visions of my future self! It's horrible!",
-						"Good. Now that the council is assembled the meeting can begin.",
-						"Listen! I only have so much time before I'm ripped away. The secret behind the gas giants are...",
-						"Das ist nicht deutschland. Das ist nicht akzeptabel!!!",
-						"I've come from the future to warn you about eigenstasium! Oh no! I'm too late!",
-						"You fool! You took too much eigenstasium! You've doomed us all!",
-						"Don't trust any bagels you see until next month!",
-						"What...what's with these teleports? It's like one of my Japanese animes...!",
-						"Ik stond op het punt om mehki op tafel te zetten, en nu, waar ben ik?",
-						"Wake the fuck up spaceman we have a gas giant to burn",
-						"This is one hell of a beepsky smash.",
-						"Now neither of us will be virgins!")
+					var/static/list/say_phrases = strings(EIGENSTASIUM_FILE, "lines")
```

Comment:
There's no reason to cache this, `strings()` does it for you already, using `GLOB.string_cache`